journalpump/journalpump.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016, Aiven, https://aiven.io/
+# Copyright 2015, Aiven, https://aiven.io/
```

Comment:
Copyright year is supposed to include the date of first publication, updating it every year is pointless (or eve counter productive).  Also, none of the other copyright claims were updated to match this.